ddtrace/internal/writer.py

```diff
@@ -141,7 +143,7 @@ def get_json(self):
                 )
                 return
 
-            return loads(body)
+            return json.loads(body)
```

Comment:
nit: this json change looks to be unnecessary

Reply:
oh, good catch. I did have a json usage in the original version. I'll adjust